plugins/modules/copr.py

```diff
@@ -270,25 +268,24 @@ def _enable_repo(self, repo_filename_path, repo_content=None):
         """
         if not repo_content:
             repo_content = self._download_repo_info()
+            repo_content = repo_content.rstrip('\n')
```

Comment:
I think this is a breaking change, since now `repo_content` is modified.

(Also it's only modified if it was downloaded, but not if it was passed in.)